### PR TITLE
chore: update repository for new standardization 

### PR DESCRIPTION
**Pull request by bloodf**
With the PR, we will put the boilerplate into a new process of development

### CODE REVIEW

File: packages/theme/nuxt.config.js [hunk 1]
```diff
@@ -5,8 +5,6 @@ export default {
     port: 3000,
     host: '0.0.0.0'
   },
-
-  // Global page headers: https://go.nuxtjs.dev/config-head
```
Comment (filipsobol) [inline packages/theme/nuxt.config.js:9]: Please restore all these comments, as they might be useful for integrators or end developers

**Review by andrzejewsky — APPROVED** [review]

File: packages/theme/nuxt.config.js (hunk 1 above)
Comment (bloodf) [reply packages/theme/nuxt.config.js:9]: Fixed

**Comment by bloodf** [comment]
Can you recheck @filipsobol

File: .vuestorefrontcloud/docker/docs/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM node:12 AS build
```
Comment (filipsobol) [inline .vuestorefrontcloud/docker/docs/Dockerfile:1]: If we require Node v14, then we should also use it for testing and deployment too

**Review by filipsobol — CHANGES_REQUESTED** [review]
I tried running the command from the README file, but it doesn't work for me.

```
fsobol:~/Projects/Boilerplate_copy$ grep -rl '<% INTEGRATION %>' ./ | xargs sed -i '' 's/<% INTEGRATION %>/sloth/g'
sed: can't read s/<% INTEGRATION %>/sloth/g: No such file or directory
```

**Review by filipsobol — APPROVED** [review]

**Pull request merged**